libmscore/excerpt.cpp

```diff
@@ -429,6 +429,22 @@ static void cloneTuplets(ChordRest* ocr, ChordRest* ncr, Tuplet* ot, TupletMap&
       ncr->setTuplet(nt);
       }
 
+//---------------------------------------------------------
+//   processLinkedClone
+//---------------------------------------------------------
+
+void Excerpt::processLinkedClone(Element* ne, Score* score, int strack)
```

Comment:
I'm sure the answer will present itself as I study this more later, but was it really necessary to create a whole new function here?  Or is this just refactoring?  More explanation would help.

Reply:
Well, I didn't just want to copy and paste this code. It's used twice a little further down in the bit where it checks for a fret diagram harmony.

Reply:
Got it, I missed that it is actually being used multiple places now.  I just did a very quick check before heading out.